apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerDbTypeTests.ValueConversions.cs

```diff
@@ -255,6 +270,25 @@ public static IEnumerable<object[]> GetValidValueConversions()
             yield return new object[] {"1", SpannerDbType.Numeric, "\"1\""};
             yield return new object[] {"1.5", SpannerDbType.Numeric, "\"1.5\""};
             yield return new object[] {DBNull.Value, SpannerDbType.Numeric, "null"};
+            // PgNumeric tests.
+            yield return new object[] { PgNumeric.NaN, SpannerDbType.PgNumeric, "\"NaN\"" };
```

Comment:
There;s a Quote method that I think will escape the quotations, see a few lines up.